pkg/mailer/mailer.go

```diff
@@ -39,16 +39,21 @@ func NewMessageFrom(to []string, from, subject, htmlBody string) *Message {
 
 	contentType := "text/html"
 	body := htmlBody
-	if setting.MailService.UsePlainText {
+	switchedToPlaintext := false
+	if setting.MailService.UsePlainText || setting.MailService.AddPlainTextAlt {
 		plainBody, err := html2text.FromString(htmlBody)
 		if err != nil {
 			log.Error(2, "html2text.FromString: %v", err)
 		} else {
 			contentType = "text/plain"
 			body = plainBody
+			switchedToPlaintext = true
 		}
 	}
 	msg.SetBody(contentType, body)
+	if switchedToPlaintext && setting.MailService.AddPlainTextAlt && !setting.MailService.UsePlainText {
+		msg.AddAlternative("text/html", htmlBody)
```

Comment:
From what I read on this line, seems like it is "Support HTML alt for plaintext emails"?

Reply:
That's a technical detail of how to setup the MIME message, but semantically it's the other way around. Mail clients will show the HTML as first priority, and the plaintext is just a backup.

Reply:
According to https://godoc.org/gopkg.in/gomail.v2#Message.AddAlternative

Reply:
I'll add a code comment